pkgs/development/libraries/vcg/default.nix

```diff
@@ -14,7 +14,11 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p $out/include
     cp -r vcg wrap $out/include
-    find $out -name \*.h -exec sed -i 's,<eigenlib/,<eigen3/,g' {} \;
+    find $out -name \*.h -exec sed -i"" \
```

Comment:
Supposedly necessary to get in-place `sed` with multiple expressions from a `find` to work